core/src/test/scala/org/apache/spark/storage/ShuffleBlockFetcherIteratorSuite.scala

```diff
@@ -1693,4 +1700,42 @@ class ShuffleBlockFetcherIteratorSuite extends SparkFunSuite with PrivateMethodT
       ShuffleBlockId(0, 5, 2), ShuffleBlockId(0, 6, 2)))
   }
 
+  test("logging of slow block fetches") {
```

Comment:
Can you prefix with the ticket number like: "SPARK-36215 logging of slow block fetches"

Reply:
Fixed